examples/phi2/phi2.py

```diff
@@ -159,19 +170,16 @@ def main(raw_args=None):
                     pass_flow.remove("perf_tuning")
 
         if "cuda" in model_type:
-            template_json["systems"]["local_system"]["config"]["accelerators"][0]["device"] = "gpu"
-            template_json["systems"]["local_system"]["config"]["accelerators"][0]["execution_providers"] = [
-                "CUDAExecutionProvider"
-            ]
+            update_accelerator(template_json, "gpu")
+
         if "cpu" in model_type:
-            # no need to set device for CPU since default it is CPU
-            template_json["systems"]["local_system"]["config"]["accelerators"][0]["execution_providers"] = [
-                "CPUExecutionProvider"
-            ]
-    if args.optimum_optimization or (args.finetune_method and not args.model_type):
+            update_accelerator(template_json, "cpu")
+
+    if args.optimum_optimization or (args.finetune_method and not args.model_type) or args.slicegpt:
```

Comment:
could be simplified as `args.optimum_optimization or not args.model_type`? @trajepl